ideascube/blog/tests/test_search.py

```diff
@@ -12,16 +12,19 @@ def test_nothing_is_indexed_without_any_fixture():
     assert Content.objects.count() == 0
 
 
+@pytest.mark.usefixtures('cleansearch')
```

Comment:
I prefer to have it explicit.
On top of the performance point, it let remind future developer that this test especially need
a clean search database